modules/fcitx5/template.nix

```
@@ -0,0 +1,234 @@
+{ colors }:
+with colors;
+{
+  Metadata = {
+    Name = "Stylix";
+    Version = 0.1;
+    Author = "sanweiya and ontake (make-42)";
+    Description = "Stylix fcitx5 theme based on the mellow-themes by sanweiya";
+  };
+  InputPanel = {
+    NormalColor = base05;
+    HighlightCandidateColor = base04;
```

I feel like base06 would make more sense here, having the selected option being dimmer than the others feels wrong
```suggestion
    HighlightCandidateColor = base06;
```